usr/share/rear/backup/NETFS/default/500_make_backup.sh

```diff
@@ -292,7 +292,7 @@ case "$(basename $BACKUP_PROG)" in
             if (( $backup_prog_rc == 1 )); then
                 LogUserOutput "WARNING: $prog ended with return code 1 and below output:
   ---snip---
-$(grep '^tar: ' "${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log" | sed -e 's/^/  /' | tail -n3)
+$(sed -n -e '/^tar: .*\(socket ignored\|Removing leading\)/d;/^tar:/s/^/  /p' "${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log" | tail -n3)
```

Comment:
In any case an explanatory comment is needed
that describes the reasoning behind that `sed` command,
cf. the section "Code must be easy to understand" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
`sed` can perform both the search for lines starting with "tar:" and the replacement. Since it is invoked with _-n_ it does not print anything unless explicitly stated (the _p_ option in the substitution. The alternative would be to use two `grep` commands, e.g.
```
grep '^tar: ' "${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log" | grep -F -v -e 'socket ignored' -e 'Removing leading' | sed -e 's/^/  /' | tail -n3
```
This is less readable than the `sed` expression, in my opinion.